frontend/components/dashboard/DashboardValidatorManagementModal.vue

```diff
@@ -44,10 +44,9 @@ const selectedGroup = ref<number>(-1)
 const {
   addEntities,
   dashboardKey,
-  isPublic: isPublicDashboard,
+  isGuest,
```

Comment:
you could think about making things clearer by changing this to `isGuestDashboard`, as the `dashboardKey` is not a `guest`, which is confusing

Reply:
fixed, applied also to other dashboard types (`isShared` and `isPrivate`)

Reply:
🤝